src/Symfony/Component/Serializer/Mapping/Loader/XmlFileLoader.php

```diff
@@ -68,7 +68,13 @@ public function loadClassMetadata(ClassMetadataInterface $classMetadata): bool
                 }
 
                 if (isset($attribute['serialized-name'])) {
```

Comment:
Not know how to mark attribute deprected. How can I make it ?

Reply:
You can use `trigger_deprecation('symfony/serializer', '6.2', 'My deprecation message')` I guess :slightly_smiling_face:

Reply:
Finally, In order to improve DX I accept two syntax but not at same time

```xml
<attribute name="foo" serialized-name="baz" />  <-- Like before -->
<attribute name="bar" serialized-name="qux" />
<attribute name="corge">
    <serialized_name name="nameOne">  <-- Or new syntax in order to change name per group -->
        <group>groupA</group>
        <group>groupB</group>
    </serialized_name>
    <serialized_name name="nameTwo">
        <group>groupC</group>
    </serialized_name>
</attribute>
<attribute name="forbid" serialized-name="baz"> <-- Forbiden (attribute and element at same time)-->
    <serialized_name name="nameOne"> 
        <group>groupA</group>
        <group>groupB</group>
    </serialized_name>
    <serialized_name name="nameTwo">
        <group>groupC</group>
    </serialized_name>
</attribute>
        ```